include/myst/syscallext.h

```diff
@@ -30,6 +30,8 @@ typedef enum
     SYS_myst_get_exec_stack_option = 2018,
     SYS_myst_interrupt_thread = 2019,
     SYS_myst_pre_launch_hook = 2020,
+    SYS_myst_debug_malloc_check = 2021,
+    SYS_myst_mcheck = 2022,
```

Comment:
What's SYS_myst_mcheck for?

Reply:
It is defined as part of ``myst_mman_mcheck()`` and it checks whether a memory range has certain permissions. It is used by the backtrace function.